packages/ionic/src/commands/deploy/manifest.ts

```diff
@@ -31,10 +32,32 @@ export class DeployManifestCommand extends Command {
       throw new FatalException(`Cannot run ${input('ionic deploy manifest')} outside a project directory.`);
     }
 
-    const buildDir = path.resolve(this.project.directory, 'www'); // TODO: this is hard-coded
+    let buildDir: string;
+    if (this.project.getIntegration('capacitor') !== undefined) {
```

Comment:
Don't you already do this logic in `deploy/core.ts`? Can you reuse it? If you need to, you can create another base class for the deploy commands that they all inherit from.